bin/scss-to-json

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var minimist = require('minimist');
+
+var scssToJson = require('./../main.js');
```

Comment:
Since the package.json specifies the main file, this should probably just be `require('../')`